docs/about/license.md

```diff
@@ -91,8 +91,7 @@ additional licenses. Please review individual repositories for more information.
 
 Open sourced ROCm components are released via public GitHub
 repositories, packages on [https://repo.radeon.com](https://repo.radeon.com) and other distribution channels.
-Proprietary products are only available on [https://repo.radeon.com](https://repo.radeon.com). Currently, only
-one component of ROCm, `rocm-llvm-alt` is governed by a proprietary license.
+Proprietary products are only available on [https://repo.radeon.com](https://repo.radeon.com).
```

Comment:
I dont have enough context to understand why it's important to mention that proprietary products are only available on repo.radeon.com if ROCm has no proprietary products. IMO, But, the information is correct as is, so I'm ok with it.